pkg/dispatcher/cache/event_handlers.go

```diff
@@ -18,24 +18,37 @@ package cache
 
 import (
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
 	clusterv1alpha1 "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
 	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
 	schedulingv1 "k8s.io/api/scheduling/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/util/retry"
 	"k8s.io/klog/v2"
 	"volcano.sh/apis/pkg/apis/scheduling"
 	"volcano.sh/apis/pkg/apis/scheduling/scheme"
 	schedulingapi "volcano.sh/volcano/pkg/scheduler/api"
 	volcanoapi "volcano.sh/volcano/pkg/scheduler/api"
 
+	"volcano.sh/volcano-global/api/datadependency/v1alpha1"
 	"volcano.sh/volcano-global/pkg/dispatcher/api"
 	"volcano.sh/volcano-global/pkg/utils"
 	"volcano.sh/volcano-global/pkg/workload"
 )
 
+const (
+	PlacementInjectedAnnotation = "datadependency.volcano.sh/placement-injected"
```

Comment:
These common consts should only be defined in one common plublic file, I just finded that you have also define this annotation key in `pkg/controller/datadepency/handler.go`